src/GitHub.VisualStudio/Menus/MenuBase.cs

```diff
@@ -46,6 +46,24 @@ protected MenuBase(IGitHubServiceProvider serviceProvider)
             apiFactory = new Lazy<ISimpleApiClientFactory>(() => ServiceProvider.TryGetService<ISimpleApiClientFactory>());
         }
 
+        protected ILocalRepositoryModel GetRepositoryByPath(string path)
+        {
+            try
+            {
+                if (!string.IsNullOrEmpty(path))
+                {
+                    var repo = ServiceProvider.TryGetService<IGitService>().GetRepository(path);
+                    return new LocalRepositoryModel(repo.Info.WorkingDirectory.TrimEnd('\\'));
```

Comment:
Thank you for your comment!

This workaround is not smart. Visual Studio's Git Provider returns directory path ends **without** "`\`", however when calling `IGitService.GetRepository(LibGit2Sharp)` with a full path of the document returns a working dir path ends **with** "`\`".

I couldn't decide where to be consistent as the best. I think that should fix as you say.